### ISSUE TITLE
Got response "<pending>" from README.md snippet

### ISSUE THREAD

**Issue by pomcho555** [node-fetch/node-fetch#1009]
This issue may be `bug report`, but I can't decide which is better. 
I'm afraid, I just post this as **Support or Usage Question**.

I ran **Buffer** snippet from README.md. I got `Promise { <pending> }` from `cosole.log`. I think it's not an appropriate output.

So, I just add `await` to ahed of `fileType.fromBuffer(buffer)`, then I got a proper output: `{ ext: 'png', mime: 'image/png' }`
This is because, I suggest we should fix README for new-comers who use this module.

**Example Code**

```js
const fetch = require('node-fetch');
const fileType = require('file-type');

(async () => {
	const response = await fetch('https://octodex.github.com/images/Fintechtocat.png');
	const buffer = await response.buffer();
	const type = fileType.fromBuffer(buffer)
	
	console.log(type);
})();
```

**Expected behavior, if applicable**

I wanna retrieve processed output.

### Your Environment

| software         | version
| ---------------- | -------
| node-fetch       | 2.6.1
| file-type.        | 16.0.1
| node             | v12.19.0
| npm              | 6.14.8
| Operating System | macOS Catalina(10.15.7)

**Additional context/Screenshots**

Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by tekwiz** [comment]
@pomcho555 Thanks for catching this.  The snippet in the README.md is missing `await` on `fileType.fromBuffer(buffer)`.  It should be:
```js
const fetch = require('node-fetch');
const fileType = require('file-type');

(async () => {
	const response = await fetch('https://octodex.github.com/images/Fintechtocat.png');
	const buffer = await response.buffer();
	const type = await fileType.fromBuffer(buffer)
	console.log(type);
})();
```

**Comment by pomcho555** [comment]
Thanks for handling this issue. If you need me, I'd love to join the review.
Also, I will close this issue after we fix #1021.
